donjo-app/models/Analisis_indikator_model.php

```diff
@@ -141,11 +143,6 @@ public function insert()
 		if ($this->analisis_master_model->is_analisis_sistem($this->session->analisis_master)) return;
 
 		$data = $this->validasi_data($this->input->post());
-		if ($data['id_tipe'] != 1)
-		{
-			$data['act_analisis'] = 2;
-			$data['bobot'] = 0;
-		}
```

Comment:
Ini diperlukan, karena seperti dijelaskan, hanya Pilihan Tunggal yg diberi bobot and aksi analisis. Masukkan ini ke `validasi_data()`